tests/test_branch_command.rs

```diff
@@ -196,17 +196,17 @@ fn test_branch_delete_glob() {
 
     // The deleted branches are still there
     insta::assert_snapshot!(get_branch_output(&test_env, &repo_path), @r###"
-    bar-2: 6fbf398c2d59 commit
+    bar-2: 6fbf398c2d59 (empty) commit |bar-2 foo-1@origin foo-3@origin foo-4@origin
```

Comment:
Maybe you've noticed, but `branches` keyword in `jj branch list` is redundant and seems rather confusing.

Reply:
Good catch, thanks! I guess for any of this to work I'd have to create at least one more template.